dotcom-rendering/src/server/lib/logging.ts

```diff
@@ -59,7 +65,7 @@ const enableLog4j = {
 		fileAppender: {
 			type: 'file',
 			filename: logLocation,
-			maxLogSize: 10000,
+			maxLogSize: '5M',
```

Comment:
Noticed that our log files are rotated every 10KB? That seemed far too small as I noticed even during local testing the log file being rotated every 5-10 requests.